src/js/course.ts

```diff
@@ -470,6 +491,17 @@ export function isClosed(c: CourseV3) {
   return c.courseEnrollmentStatus === "closed";
 }
 
+export function getCourseArea(c: CourseV3): CourseArea {
+  const code = c.courseCode.split(" ")[0];
```

Comment:
I actually think I'd prefer using regexp here, since there's technically no API-side guarantee that the course code contains a space between the department and number (e.g. `CSCI 005` instead of `CSCI005`).  Technically, it doesn't guarantee that course codes begin with their department at all, but using a regexp like `/^[A-Z]+/` would be a little more forgiving than simply splitting on a space.  Ideally, what we would want to do is add a `department` field to the API response spec.

Reply:
Makes sense. Changed to regexp matching instead.